kbase/narrative
# DATAUP-643 multiple parameter set warning

# Description of PR purpose/changes

This adds a warning banner to the top of bulk import cells if an XSV file has multiple non-unique, non-file parameters. For example, if an assembly file looks like:

```
FASTA file path, Assembly object name, Assembly Type, Minimum contig length
first_file.fasta,  first_object,  draft isolate, 1000
second_file.fasta, second_object, draft isolate, 1000
```
there would be no warning, as the parameters that aren't input files or output names are identical across all rows. If they were not identical, then the first row's params would be used, and there would be a yellow warning banner:
![Screen Shot 2022-02-28 at 10 57 41 PM](https://user-images.githubusercontent.com/2152243/156102129-4c3f09c5-3230-4481-986b-66558d5484fb.png)

I also found a bug where the widgets in the configure tab weren't being unloaded properly, which might cause memory issues, especially with large amounts of rows.

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/DATAUP-643

# Testing Instructions
* Details for how to test the PR:
* Use import spec files with: 
* 1 row - no warning
* multiple rows of unique params - no warning
* multiple rows of different params - warning
* mix and match with different types - warning should stick to only the appropriate file type tab

# Dev Checklist:


# Updating Version and Release Notes (if applicable)

- [n/a] [Version has been bumped](https://semver.org/) for each release
